x-pack/plugins/transform/public/alerting/transform_health_rule_type/register_transform_health_rule.ts

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { lazy } from 'react';
+import { i18n } from '@kbn/i18n';
+import { TRANSFORM_RULE_TYPE } from '../../../common';
+import type { TransformHealthRuleParams } from '../../../common/types/alerting';
+import type { AlertTypeModel } from '../../../../triggers_actions_ui/public';
+
+export function createTransformHealthRuleType(): AlertTypeModel<TransformHealthRuleParams> {
+  return {
+    id: TRANSFORM_RULE_TYPE.TRANSFORM_HEALTH,
+    description: i18n.translate('xpack.transform.alertingRuleTypes.transformHealth.description', {
+      defaultMessage: 'Alert when transform jobs experience operational issues.',
+    }),
+    iconClass: 'bell',
+    documentationUrl(docLinks) {
+      return docLinks.links.transforms.alertingRules;
+    },
+    alertParamsExpression: lazy(() => import('./transform_health_rule_trigger')),
+    validate: (alertParams: TransformHealthRuleParams) => {
+      const validationResult = {
+        errors: {
+          includeTransforms: new Array<string>(),
+        } as Record<keyof TransformHealthRuleParams, string[]>,
+      };
+
+      return validationResult;
+    },
+    requiresAppContext: false,
+    defaultActionMessage: i18n.translate(
+      'xpack.transform.alertTypes.transformHealth.defaultActionMessage',
+      {
+        defaultMessage: `[\\{\\{rule.name\\}\\}] Transforms health check result:
+\\{\\{context.message\\}\\}
+\\{\\{#context.results\\}\\}
+  Transform ID: \\{\\{transform_id\\}\\}
```

Comment:
Is there anything we can do about indentation when there is more than 1 transform in the notification - the second ID row is indented:

![image](https://user-images.githubusercontent.com/7405507/133598720-19da056d-0c2b-4e35-b11f-82a0b0e827bc.png)

I wonder if it is worth indenting the rows beneath transform ID, to make it clearer which info belongs with each transform?

Reply:
![image](https://user-images.githubusercontent.com/5236598/133615808-a0600ec5-23af-4f7c-8aa4-8bc36ecf2b8a.png)
Added a link break in https://github.com/elastic/kibana/pull/112277/commits/b7c5e396a587a9b82b3e06ca45c62472d349e1f7